src/app/home/home.component.html

```diff
@@ -9,3 +9,6 @@ <h1>
     <img src="/assets/images/Header_bg.png" alt="">
   </div>
 </div>
+<div style="height:1000px;position:relative;"></div>
```

Comment:
Don't give inline styles @live-wire and height of 1000px why? Any reason?

Reply:
Hey @aka-jain . I didn't want you to review this PR yet 🙈 It is still a Work in progress. I'll update you and this PR when Shivani hands in the final footer design 😄

Reply:
Ah, my bad, sorry man!